app/controllers/support_interface/countries_controller.rb

```diff
@@ -61,6 +61,7 @@ def country_params
         :region_names,
         :sanction_information,
         :status_information,
+        :eligibility_route,
```

Comment:
I think we can remove `eligibility_skip_questions` from this list as we're not passing it through to the controller anymore.